pkg/config/integration_config.go

```diff
@@ -169,6 +170,8 @@ func validateProcessingRules(rules []LogsProcessingRule) ([]LogsProcessingRule,
 		case MASK_SEQUENCES:
 			rules[i].Reg = regexp.MustCompile(rule.Pattern)
 			rules[i].ReplacePlaceholderBytes = []byte(rule.ReplacePlaceholder)
+		case MULTILINE:
+			rules[i].Reg = regexp.MustCompile("^" + rule.Pattern)
```

Comment:
1. Could we add a quick test on config being properly parsed?
2. Do we want to add the `^`? What'd be the behavior if people add it too? Should we dedupe it? If so, those are worth tests also (Opened questions)
3. What happens when the regex doesn't compile?

Reply:
Agreed for 1 and 2.
For 3, `logs-agent` crashes if the `regex` does not compile. IMO, `logs-agent` should not run if he can't process logs, we could recover and `os.Exit()` safely, WDYT ?

Reply:
for 3 I think we're fine for now. We'll see later if this is a problem, as it will be a problem for MASK_SEQUENCES too